develop-guide/writeblitz/writeblitz.rst

```diff
@@ -177,16 +164,32 @@ exists in the output. You also, have the option to check if a specific
 
 Both include and exclude keywords are optional to use.
 
+configure
+_________
+
+The `configure` action is used to configure the device.
+
+.. code-block:: YAML
+
+    - configure: # ACTION
+        device: device_name
+        command: |
+            router bgp 65000
+            shutdown
+
 .. note::
 
-    The all the arguments that can be sent to the device while executing a command on a device (specified in this `link
-    <http://wwwin-pyats.cisco.com/cisco-shared/unicon/latest/user_guide/services/generic_services.html#execute>`_.)
-    and while configuring a device (specified in this `link
-    <http://wwwin-pyats.cisco.com/cisco-shared/unicon/latest/user_guide/services/generic_services.html#configure>`_.) 
-    can be used like below example.
+    You can apply additional arguments to commands ``execute`` and ``configure``.
```

Comment:
why isnt execute with the execute.

it makes no sense to show execute in the configure section

Reply:
They were back to back actions, and technically same feature, thought would be more efficient to have them together. Changing to your way

Reply:
Done